example/src/main/java/com/example/MainActivity.java

```diff
@@ -117,13 +127,51 @@ public void trackPlay(View view) {
                 90
         );
         // NOTE: For videos embedded in an article, "url" should be the URL for that article.
-        ParselyTracker.sharedInstance().trackPlay("http://example.com/app-videos", null, metadata, null);
+        ParselyTracker.sharedInstance().trackPlay("http://example.com/app-videos", "", metadata, null);
 
     }
 
     public void trackPause(View view) {
         ParselyTracker.sharedInstance().trackPause();
     }
 
-    public void trackReset(View view) {ParselyTracker.sharedInstance().resetVideo(); }
+    public void trackReset(View view) {
+        ParselyTracker.sharedInstance().resetVideo();
+    }
+
+    private boolean engagementIsActive() {
```

Comment:
Are these private API calls really useful in the demo app? Meaning: does it make sense to show this info given it's not going to be available for clients?
I get that the intention might be just showing some debug info, but if so, perhaps good to at least add a comment highlighting that these are private methods and not intended to be used by users in general? 🤔

Reply:
> Are these private API calls really useful in the demo app?

I think yes - it might be helpful to see how the internal parameters of the SDK behave after certain actions to see how it works. Especially, as things happen here after intervals 10+ seconds.

But I see that it's the "example" app, so it should be giving a good example of the integration. I agree that in the current form it could be misleading.

In 003d6fb760f692812ca21f612324610cb24bf5e0 I've extracted all calls to a separate class and added comments. WDYT?

Reply:
> But I see that it's the "example" app, so it should be giving a good example of the integration
🎯 Exactly!

> In https://github.com/Parsely/parsely-android/commit/003d6fb760f692812ca21f612324610cb24bf5e0 I've extracted all calls to a separate class and added comments. WDYT?

This is a good compromise IMO, and it makes it clear to clients that this is just for debugging 👍 Thanks for taking the time to address it!